lib/modules/apostrophe-images/lib/cursor.js

```diff
@@ -40,6 +40,65 @@ module.exports = {
       }
     });
 
+    self.addFilter('orientation', {
+      finalize: function() {
+        const orientation = self.get('orientation');
+        if (!orientation) {
+          return;
+        }
+
+        self.and(getCriteria(orientation));
+
+        function getCriteria(orient) {
+          if (orient === 'landscape') {
+            return {'attachment.landscape': true};
```

Comment:
Consistent spacing.

Reply:
Newlines for objects. (eslint will catch these too I think.)

Reply:
No eslint error neither for spacing nor for new line. Ok